.github/workflows/ghcr-release.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'A version number'
```

Comment:
should there be an actual version number in  here?

Reply:
You will be asked for a version number every time you run the action. So if you want to release `0.1.7` you would put that in the field. You can also enter `latest` to update the latest tag in ghcr.